content/docs/transforms/examples.md

```diff
@@ -0,0 +1,283 @@
+---
+metaTitle: "Example Transforms"
+metaDescription: "Example transform scripts for automating qri dataset updates"
+weight: 3
+---
+
+import InfoBlock from '../../../src/components/InfoBlock.js'
+
+<InfoBlock>
+  This documentation is for qri CLI (command line) only. Composing Transform scripts on Desktop is a work in progress.
+</InfoBlock>
+
+Transforms allow you to bind data with code in Qri, providing a quick means to automate the creation of new dataset versions from websites, APIs, and _other qri datasets_.
+
+To get started with transforms, check out [Transforms Qruickstart](https://qri.io/docs/transforms/quickstart).
+
+Below are several examples of transform scripts that demonstrate various approaches to automating qri datasets.  If you have a transform script that you think might help others, consider [opening a pull request](https://github.com/qri-io/website/pulls) to add it here.
+
+-- --
+
+## Count NYC 311 Complaints By Boro (Spatial Join and Aggregation)
+
+### Overview
```

Comment:
Can we drop the `### Overview` and `### Script` headers? They have the same font size as the section titles when rendered, making the page a little harder to read

Reply:
For sure, was planning to have more sections but ended up with just one intro paragraph and the code snippet.